csharp/extractor/Semmle.Extraction/Entities/Assembly.cs

```diff
@@ -30,7 +30,7 @@ public override void Populate(TextWriter trapFile)
         {
             if (assemblyPath != null)
             {
-                trapFile.assemblies(this, File.Create(Context, assemblyPath), assembly.ToString(),
+                trapFile.assemblies(this, File.Create(Context, assemblyPath), assembly.ToString() ?? "",
```

Comment:
Should this be `assembly.ToString()!` instead?

Reply:
Well, technically `IAssemblySymbol.toString()` could return `null`, in which case we would rather use `""` than risk propagating it.